infra/aws/terraform/prow-build-cluster/README.md

```diff
@@ -243,3 +243,63 @@ make destroy
 
 If you want to remove roles used for EKS creation go to `../iam/<aws_account_name>` and run `terraform destroy` command there.
 
+
+## GitOps
+
+To sync state from this Git repo into EKS cluster we use [FluxCD](https://fluxcd.io/).
+
+FluxCD has a dedicated [CLI tool](https://fluxcd.io/flux/installation/#install-the-flux-cli). We use it iniside our scripts to generate and monitor syncs.
+
+FluxCD resources stored inside the `./resources` directory and have been generated with use of `./hack/flux-update.bash`. The script prepares manifests for the [GitOps Tool Kit](https://fluxcd.io/flux/components/) and generates [Sources](https://fluxcd.io/flux/components/source/), [Kustomizations](https://fluxcd.io/flux/components/kustomize/kustomization/) and [Helm Releases](https://fluxcd.io/flux/components/helm/helmreleases/).
+
+The `flux-system` namespace contains all GitOps Tool Kit componenets as well as all Flux Sources and Kustomizations. Helm Releases ought to be deployed in the same namespaces as manifests they create. As a convention, Flux Helm Releases have to be prefixed with `flux-hr-`, Kustomizations with `ks-` and sources should contain `-source-` part. These are our internal conventions and are used for discovery process in our scripts:
+
+- Installing Flux:
+    ```bash
+    make flux-install
+    ```
+
+- Deploying Sources:
+    ```bash
+    make flux-apply-sources
+    ```
+
+- Deploying Kustomizations:
+    ```bash
+    make flux-apply-kustomizations
+    ```
+
+- Deploying Helm Releases:
+    ```bash
+    make flux-apply-helm
+    ```
+
+### Accessing Flux
+
+In order to access FluxCD, all you need is kubeconfig with broad cluster permissions and Flux CLI installed locally.
```

Comment:
I would extend this with a note that you:

- only need this to administer the FluxCD instance
- can reach out SIG K8s infra over Slack if you need anything being done in regards to the Flux installation